server/helper.js

```diff
@@ -0,0 +1,42 @@
+const errorToJson = (err) => {
+  return {
+    error: (process.env.NODE_ENV === 'production') ? {} : err,
+    message: err.message
+  };
+};
+
+const isOwner = (req, res, next) => {
+  let id = req.params.id;
+  if (id === req.user._id.toString()) {
+    next();
+  } else {
+    res.status(400).json(errorToJson(new Error('You don\'t have permissions over this user')));
+  }
+};
+
+const isAdmin = (req, res, next) => {
+  if(req.user.isAdmin) {
+    next();
+  } else {
+    res.status(403).json(errorToJson(new Error('You\'re not an admin')));
+  }
+};
+
+// Remove email, linkedin ID, hash, salt, isAdmin
+const anonymiseUser = (user) => {
+  return {
+    id: user._id,
```

Comment:
Could we change this to also use `_id` to stay consistent with all other items that are being returned.
```suggestion
    _id: user._id,
```